### ISSUE TITLE
Tailwind + Vue + karma

### ISSUE THREAD

**Issue by george-silva** [tailwindcss/discuss#152]
Hi!

I'm trying to use vue (vue-cli with webpack template, basically) with tailwindcss. Everything works out fine, except my tests using karma + mocha, which gives me this error:

```
12 06 2018 00:42:41.429:INFO [karma]: Karma v1.7.1 server started at http://0.0.0.0:9876/
12 06 2018 00:42:41.432:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
12 06 2018 00:42:41.437:INFO [launcher]: Starting browser PhantomJS
12 06 2018 00:42:41.829:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: Connected on socket drO3nZzaT4uQmbtiAAAA with id 92698283
PhantomJS 2.1.1 (Linux 0.0.0) ERROR
  SyntaxError: Unexpected token '>'
  at webpack:///node_modules/tailwindcss/lib/plugins/container.js:10:0 <- index.js:15600

PhantomJS 2.1.1 (Linux 0.0.0): Executed 0 of 0 ERROR (0.288 secs / 0 secs)


PhantomJS 2.1.1 (Linux 0.0.0): Executed 0 of 0 ERROR (0.001 secs / 0 secs)
```

My webpack config is basically unchanged from the one that comes with vue-cli. I did added

```
      // {
      //   test: /\.css$/,
      //   exclude: /node_modules/,
      //   include: [resolve('src')],
      //   use: [
      //     {
      //         loader: 'style-loader',
      //     },
      //     {
      //         loader: 'css-loader',
      //         options: {
      //             importLoaders: 1,
      //         }
      //     },
      //     {
      //         loader: 'postcss-loader'
      //     }
      //   ]
      // },
```

But as you can see, this is commented and NOT in place (looks llike webpack template generates bunch of style loaders and injects them).

In summary: tailwind works fine in dev and prod. But I can't test anything because my tests fail with the error message above.

I'm importing my `styles.css` like so:

```
<template>
<router-view></router-view>
</template>

<script>
export default {
  name: 'App'
}
</script>
<style>
@import './assets/css/styles.css'
</style>
```

**Comment by adamwathan** [comment]
What version of node are you using? Try upgrading to make sure you are on
at least 6.9 and see if that helps; that’s the target we compile for.

**Comment by adamwathan** [comment]
Seems like the problem is with PhantomJS:

https://stackoverflow.com/questions/48580544/phantom-js-doesnt-support-arrow-function

Not sure what the best solution is here; I feel like PhantomJS isn't really widely used anymore. I'm surprised that it's even trying to run the Tailwind JS in a PhantomJS process though, sort of expected that stuff would be pre-compiled before any tests try to run so maybe there's a way to configure it to work like that?

**Comment by george-silva** [comment]
Good catch. i've googled for a long time about karma, postcss, etc.

The idea is to use chrome headless? I really didn't touched anything on the test side and I'm kind new to the whole new JS thing (JS moves soooo fast).

Let me try with headless chrome and I'll report back.

**Comment by george-silva** [comment]
Jeez! You've killed it!

The issue is with **PhantomJS**. I've installed `karma-chrome-launcher` and switched the browsers array from: 

`browsers: ['PhantomJS']`

to

`browsers: ['ChromeHeadless']`
